### DISCUSSION TITLE
Dear teachers, I need some help: when I start drawing pictures with disco diffusion, my account always pops up human-machine verification, but there is no picture that needs to be verified, what should I do?

### DISCUSSION THREAD

**Discussion by jing61606** [community/community#40333] [GitHub Education]
### Select Topic Area

Question

### Body

尊敬的各位前辈老师，我需要求助一下：我准备开始用disco diffusion画图时，我的账号总是弹出来人机验证，但是又没有弹出来需要人机验证的图片，（人机验证勾选验证出现的是：您的计算机或者网络可能在发送自动查询内容，为了保护我们的用户，我们目前无法处理您的请求，如需了解更多详情，请访问我们的帮助页面）请问要怎么办？ 我的账号总是三天两头画不起，拜托了请帮忙解答一下，万分感激！
Dear senior teachers, I need some help: when I start drawing with disco diffusion, my account always pops up human-computer verification, but there is no picture that needs human-computer verification. What appears is: Your computer or network may be sending automated queries. To protect our users, we are currently unable to process your request. For more details, please visit our help page) What should I do? My account is always unable to draw in two or three days, please help me to answer, thank you very much!

**Comment by jing61606** [comment]
![d7b0d13f2bcf5f0e3eb1f28d2bbcec7](https://user-images.githubusercontent.com/119399098/204479882-e17e45cd-b9e7-48cb-a6c3-161a149daba2.png)
![4fb37ed097af3d7664d00fa32b641d5](https://user-images.githubusercontent.com/119399098/204479949-9a5a8bbc-154a-44c6-96a4-6cc72e968ef7.png)
总是出现这个，然后就不能画图了，请问这个需要怎么解决呢？
This always appears, and then I can’t draw pictures. How can I solve this problem?

**Comment by ossanna16** [comment]
Hi there @jing61606 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
